pkg/auth/policy_store_test.go

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package auth
+
+import (
+	"testing"
+
+	"github.com/golang-collections/collections/set"
+
+	"kmesh.net/kmesh/api/v2/workloadapi/security"
+)
+
+// TestConcurrency tests concurrency safety of policyStore, success if no panic
+func TestConcurrency(t *testing.T) {
```

Comment:
This test is less meaningful, we can the methods of policyStore and check the expected result